build/InputDirectories.props

```diff
@@ -4,6 +4,7 @@
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
     <MSBuildImportsDir>$(RepoRoot)/resources/MSBuildImports</MSBuildImportsDir>
     <SrcDirectory>$(RepoRoot)/src</SrcDirectory>
+    <ProjectsSrcDirectory>$(RepoRoot)/src</ProjectsSrcDirectory>
```

Comment:
This is a duplicate property of the one directly above it.  Why do we need 2 duplicate global properties?  It makes it hard to figure out which one I should use.